spec/features/step_definitions/connectivity_steps.rb

```diff
@@ -2,12 +2,12 @@
 
 step 'I am consuming' do
   set_event_routing
-  initialize_queue
+  set_event_definitions
+  RabbitFeed::ConsumerConnection.instance # Bind the queue
```

Comment:
I see what you mean - not sure I'll change anything now - I'd have to think about it.